examples/fscloud/main.tf

```diff
@@ -70,6 +70,9 @@ module "cbr_account_level" {
   # Demonstrates how zone creation will be skipped for these two service references ["user-management", "iam-groups"]
   skip_specific_services_for_zone_creation = ["user-management", "iam-groups"]
 
+  # This example scope the zones to us-south region.
+  location = "us-south"
```

Comment:
I would just not scope in the example here as I think this will lead to confusion

Reply:
ok, removed.